types/lib/utils/gestures.d.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   lib/utils/gestures.html
+ */
+
+/// <reference path="boot.d.ts" />
+/// <reference path="async.d.ts" />
+/// <reference path="debounce.d.ts" />
+
+declare namespace Polymer {
+
+  /**
+   * Module for adding listeners to a node for the following normalized
+   * cross-platform "gesture" events:
+   * - `down` - mouse or touch went down
+   * - `up` - mouse or touch went up
+   * - `tap` - mouse click or finger tap
+   * - `track` - mouse drag or touch move
+   */
+  namespace Gestures {
+
+
+    /**
+     * Finds the element rendered on the screen at the provided coordinates.
+     *
+     * Similar to `document.elementFromPoint`, but pierces through
+     * shadow roots.
+     */
+    function deepTargetFind(x: number, y: number): Element|null;
+
+
+    /**
+     * Adds an event listener to a node for the given gesture type.
+     */
+    function addListener(node: Node|null, evType: string, handler: Function|null): boolean;
```

Comment:
All of these (and in the later methods) can not be `|null`.

Reply:
Done. Updated `@param` annotations.